.platform.app.yaml

```diff
@@ -0,0 +1,92 @@
+# This file describes an application. You can have multiple applications
+# in the same project.
+
+# The name of this app. Must be unique within a project.
+name: php
+
+# The toolstack used to build the application.
+type: "php"
+
+build:
+    flavor: "composer"
+
+# The configuration of app when it is exposed to the web.
+web:
+    # The public directory of the app, relative to its root.
+    document_root: "/_build/html"
+    index_files:
+      - index.html
+    whitelist:
+      - \.html$
+
+      # CSS and Javascript.
+      - \.css$
+      - \.js$
+      - \.hbs$
+
+      # image/* types.
+      - \.gif$
+      - \.jpe?g$
+      - \.png$
+      - \.tiff?$
+      - \.wbmp$
+      - \.ico$
+      - \.jng$
+      - \.bmp$
+      - \.svgz?$
```

Comment:
I think we can remove `tiff`, `wbmp`, `jpg`, `jpeg`, `jng` and `bmp` from this list.